pandas/core/window/rolling.py

```diff
@@ -1033,6 +1072,9 @@ def validate(self):
         else:
             raise ValueError(f"Invalid window {self.window}")
 
+        if self.method != "single":
+            raise NotImplementedError("'single' is the only supported method type.")
```

Comment:
test for this?

Reply:
Yup, here: https://github.com/pandas-dev/pandas/pull/38417/files#diff-973262cf0066942aadb7ffc25715c0ea75b3159b725383e41efed6f35de71f16R124